docs/experiment/sdks/android-sdk.md

```diff
@@ -159,6 +159,8 @@ If you use either Amplitude or Segment Analytics SDKs to track events into Ampli
         )
         ```
 
+    Note that, if you are using a custom instance name for analytics, you will need to set the same value in the `instanceName` [configuration option](#configuration) in the experiment SDK.
```

Comment:
It would be great to mention that the `instanceName` is case insensitive.

Reply:
Thanks! Updated!